notifications_utils/jinja_templates/letter_image_template.jinja2

```diff
@@ -1,3 +1,14 @@
+{% if too_many_pages %}
+  <div class='banner-dangerous bottom-gutter'>
+    <h1 class='banner-title'>
+      This letter is too long
```

Comment:
This h1 banner title more or less repeats the description.
Suggest we use:
`There is a problem`
as described in the design system: https://design-system.service.gov.uk/components/error-summary/

I'm working with the other GaaP content designer to try and make our approach to error messages more consistent across Notify, Pay and Registers.

Reply:
I strongly feel that being specific and direct is better here.

The design system has to use ‘There is a problem’ because it’s dealing, potentially, with multiple errors (so should really be ‘There are some problems’).

Reply:
I think you're right that a specific and direct error summary is better in cases where the page can only display a single error.

If we keep `This letter is too long` as the summary we could change the description to avoid the repetition, eg:

`Letters must be 10 pages or less`

Reply:
![image](https://user-images.githubusercontent.com/355079/44452216-d8ec5800-a5ed-11e8-99d6-6fd88c09c2cd.png)

2886e0d